package.json

```diff
@@ -52,7 +51,7 @@
     "gulp-tslint": "^6.1.1",
     "gulp-typescript": "^2.13.4",
     "gulp-typings": "^2.0.0",
-    "merge-stream": "^1.0.0",
+    "merge-stream": "^1.0.1",
```

Comment:
Does this need to be 1.0.1? Prefer the lowest bounds that works to avoid duplicates in npm